package.json

```diff
@@ -40,10 +40,11 @@
     "can-event-queue": "^1.0.1",
     "can-list": "^4.0.0",
     "can-log": "^1.0.0",
-    "can-map": "^4.0.1",
+    "can-map": "^4.3.5",
```

Comment:
I think we should keep backwards compatibility by checking for both `__inSetup` and the symbol, so we don’t require all dependencies to be updated with this change. Thoughts?

Reply:
I thought the intention of this cleanup - https://github.com/canjs/canjs/issues/4705, was to replace all instances of `__inSetup` with the Symbol. Unless I am misunderstanding the issue

Reply:
Reverted to not changing the semver. 👍

Reply:
I think we should update the `can-map` dependency. Since, `can-map-define` does not directly depend on `can-construct`, the version of `can-construct` installed will be the one that `can-map` installs. For this reason, now that `can-map-define` is reading the new Symbol, it needs to depend on a version of `can-map` that depends on a version of `can-construct` that is setting the Symbol.

Doing it this way should not force users to update multiple dependencies at once. If they update `can-map-define`, they will get a version of `can-construct` that works with this change.